### DISCUSSION TITLE
Would make sense to run workflows with other supported Node versions?

### DISCUSSION THREAD

**Discussion by oscard0m** [mswjs/msw#1305] [Q&A]
Would make sense to run MSW workflows with Node version in _Maintainance_ and Node version _Current_ as well? `Node 14` and `Node 18` this message was posted.

Not only for this repo but for other repos under `mswjs`

**Comment by kettanaito** [comment]
Hey, @oscard0m. We used to run CI against Node 14 and 16 but I have removed 14 in the past. We still support Node 14, so I think adding back that target is a good idea:

https://github.com/mswjs/msw/blob/362b7cbd318503e6831e4f0c3aaa036a9c131a56/package.json#L10-L12

We can add it only for the [build](https://github.com/mswjs/msw/blob/362b7cbd318503e6831e4f0c3aaa036a9c131a56/.github/workflows/ci.yml#L13) job, leaving the type checking to run on Node 16 always. What do you think? 

As to Node 18, I can't say right away that we support it. We didn't limit the `engines` range so that people could install MSW on any Node 14+ but it depends on the exact Node API that they are using, which may not be supported by MSW yet (like the global `fetch` in Node, or Undici).
